File: python/숨어있는 숫자의 덧셈 (2).py
def solution(my_string):
    answer = 0
    temp = ''
    for i in my_string:
        if i.isdigit():
            temp += i
        else:
            if temp:
                answer += int(temp)
                temp = ''
    if temp:
        answer += int(temp)
    return answer

File: java/숨어있는 숫자의 덧셈 (2).java
class Solution {
    public int solution(String my_string) {
        int answer = 0;
        for(String s : my_string.split("[a-zA-Z]+")) {
            if(!s.isEmpty()) {
                answer += Integer.parseInt(s);
            };
        }
        return answer;
    }
}